cdap-watchdog/src/main/java/co/cask/cdap/logging/serialize/LoggingEventSerializer.java

```diff
@@ -138,7 +139,7 @@ public GenericRecord toGenericRecord(ILoggingEvent event) {
     Schema schema = getAvroSchema();
     GenericRecord datum = new GenericData.Record(schema);
     datum.put("threadName", event.getThreadName());
-    datum.put("level", event.getLevel().toInt());
+    datum.put("level", event.getLevel() == null ? Level.ERROR_INT : event.getLevel().toInt());
```

Comment:
even earlier we didn't do null check for level right ? we did it for argument array but not level.